source/includes/connection-read-config.rst

```diff
@@ -0,0 +1,31 @@
+If you use :ref:`SparkConf <spark-conf>` to specify any of the previous settings, you
+can either list them individually or include them in the ``connection.uri`` setting.
+
+The following code example shows how to specify the ``connection.uri``,
+``database``, ``collection``, and ``readPreference`` settings individually in a
+``SparkConf`` configuration file:
+
+.. code:: cfg
+
+   spark.mongodb.read.connection.uri=mongodb://127.0.0.1/
+   spark.mongodb.read.database=test
+   spark.mongodb.read.collection=myCollection
+   spark.mongodb.read.readPreference.name=primaryPreferred
+
+Instead, you can specify these settings in the value of ``connection.uri``:
```

Comment:
Q: Is this the preferred/recommended way to set the settings as opposed to doing so individually? I think leading in with "Instead, you can..." makes it sound like the previous option is not as good as this one.

Reply:
pretty sure individual lines is preferred, so i put that second.